package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "clean": "rm -rf src/*/node_modules/ build/ node_modules/ src/server_manager/install_scripts/do_install_script.ts",
     "do": "bash ./scripts/do_action.sh",
-    "lint": "tslint 'src/**/*.ts' -e '**/node_modules/**'"
+    "lint": "tslint 'src/**/*.ts' -e '**/node_modules/**'",
```

Comment:
This is annoying. Can we download the binary and put it somewhere?

Something like this could be in our package.json dev install, perhaps in a script:
```bash
mkdir -p ./downloads/
curl --silent -L "https://github.com/koalaman/shellcheck/releases/download/latest/shellcheck-latest.$(uname | tr [:upper:] [:lower:]).x86_64.tar.xz" | tar x --directory ./downloads
```

I'm using `downloads/` as a suggestion. It could be in `node_modules/`, but that may upset yarn verification.

Reply:
Actually, I think it's better if we pin to a version of shellcheck. Like https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.${PLATFORM}.x86_64.tar.xz

Reply:
OK, done.  This was nontrivial, due to the need to handle various platforms, avoid repeatedly downloading the binary, check hashes, etc.

In the process of implementing this I noticed that our "husky" pre-commit hook is not running (at least not for me) so we are not actually running the linters on every commit.